File: workspace/data-model/src/main/java/de/pfiva/data/model/snips/SnipsOutput.java
package de.pfiva.data.model.snips;

import java.util.List;

public class SnipsOutput {

	private String id;
	private String input;
	private Intent intent;
	private List<Slot> slots;
	private String sessionId;
	
	private String filename;
	private String filePath;
	private String hotword;
	private String timestamp;
	
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getInput() {
		return input;
	}
	public void setInput(String input) {
		this.input = input;
	}
	public Intent getIntent() {
		return intent;
	}
	public void setIntent(Intent intent) {
		this.intent = intent;
	}
	public List<Slot> getSlots() {
		return slots;
	}
	public void setSlots(List<Slot> slots) {
		this.slots = slots;
	}
	public String getSessionId() {
		return sessionId;
	}
	public void setSessionId(String sessionId) {
		this.sessionId = sessionId;
	}
	public String getFilename() {
		return filename;
	}
	public void setFilename(String filename) {
		this.filename = filename;
	}
	public String getFilePath() {
		return filePath;
	}
	public void setFilePath(String filePath) {
		this.filePath = filePath;
	}
	public String getHotword() {
		return hotword;
	}
	public void setHotword(String hotword) {
		this.hotword = hotword;
	}
	public String getTimestamp() {
		return timestamp;
	}
	public void setTimestamp(String timestamp) {
		this.timestamp = timestamp;
	}
	@Override
	public String toString() {
		return "SnipsOutput [id=" + id + ", input=" + input + ", intent=" + intent + ", slots=" + slots + ", sessionId="
				+ sessionId + ", filename=" + filename + ", filePath=" + filePath + ", hotword=" + hotword
				+ ", timestamp=" + timestamp + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/snips/SnipsOutput.ts
import { Intent } from "./Intent";
import { Slot } from "./Slot";

export class SnipsOutput {

    private _id: string;
    private _input: string;
    private _intent: Intent;
    private _slots: Slot[];
    private _sessionId: string;

    private _filename: string;
    private _filePath: string;
    private _hotword: string;
    private _timestamp: string;

	public get id(): string {
		return this._id;
	}

	public set id(value: string) {
		this._id = value;
    }
    
	public get input(): string {
		return this._input;
	}

	public set input(value: string) {
		this._input = value;
	}

	public get intent(): Intent {
		return this._intent;
	}

	public set intent(value: Intent) {
		this._intent = value;
    }
    
	public get slots(): Slot[] {
		return this._slots;
	}

	public set slots(value: Slot[]) {
		this._slots = value;
    }
    
	public get sessionId(): string {
		return this._sessionId;
	}

	public set sessionId(value: string) {
		this._sessionId = value;
	}

	public get filename(): string {
		return this._filename;
	}

	public set filename(value: string) {
		this._filename = value;
    }
    
	public get filePath(): string {
		return this._filePath;
	}

	public set filePath(value: string) {
		this._filePath = value;
	}

	public get hotword(): string {
		return this._hotword;
	}

	public set hotword(value: string) {
		this._hotword = value;
    }
    
	public get timestamp(): string {
		return this._timestamp;
	}

	public set timestamp(value: string) {
		this._timestamp = value;
	}
}
